src/pool.jl

```diff
@@ -188,3 +205,9 @@ function Base.showerror(io::IO, err::LevelsException{T, R}) where {T, R}
     levs = join(repr.(err.levels), ", ", " and ")
     print(io, "cannot store level(s) $levs since reference type $R can only hold $(typemax(R)) levels. Use the decompress function to make room for more levels.")
 end
+
+
+# OrderedLevelsException
+function Base.showerror(io::IO, err::OrderedLevelsException)
```

Comment:
Can you add a test for this method? I think there are examples for `LevelsException`.